modules/psap-driver-toolkit-pulling.adoc

```diff
@@ -28,17 +28,29 @@ The driver-toolkit image for the latest minor release will be tagged with the mi
 +
 [source,terminal]
 ----
-$ oc adm release info 4.11.0 --image-for=driver-toolkit
+# For x86 image:
+$ oc adm release info quay.io/openshift-release-dev/ocp-release:{product-version}.z-x86_64 --image-for=driver-toolkit
+
+# For ARM image:
+$ oc adm release info quay.io/openshift-release-dev/ocp-release:{product-version}.z-aarch64 --image-for=driver-toolkit
 ----
 +
+[IMPORTANT]
+====
+ARM images are built for 64-bit Arm (AArch64) only, and are currently a Technology Preview feature. For more information about {product-title} on ARM, see xref:../release_notes/ocp-4-11-release-notes.html#ocp-4-11-release-notes[Release Notes]. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope]. 
```

Comment:
```suggestion
ARM images are built for 64-bit ARM (AArch64) only, and are currently a Technology Preview feature. For more information about {product-title} on ARM, see xref:../release_notes/ocp-4-11-release-notes.html#ocp-4-11-release-notes[Release Notes]. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope]. 
```

Reply:
Maybe there is an exception for Telco docs (?) but usually xrefs aren't allowed in OCP modules (Release Notes link).

Reply:
Hi @kcarmichael08 Thanks for your feedback. I see other links in the section so I'll ping you later to finalize this.